src/components/Reports/GoalCalculator/CalculatorSettings/Steps/SpecialIncomeStep/SpecialIncomeStepForm/SpecialIncomeStepForm.tsx

```diff
@@ -77,16 +76,14 @@ export const SpecialIncomeStepForm: React.FC<SpecialIncomeStepFormProps> = ({
       <Grid container spacing={3}>
         <Grid item xs={12}>
           <Field name="incidentIncome">
-            {() => (
+            {({ field }) => (
               <TextField
+                {...field}
                 fullWidth
                 size="small"
                 label={t('Incident Income')}
                 name="incidentIncome"
```

Comment:
You can also remove `name` here and in the other fields because it comes from `field`.